src/objective-c/tests/End2EndTest/End2EndTest/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  End2EndTest
+//
+//  Created by mxyan on 7/11/16.
+//  Copyright © 2016 Google. All rights reserved.
+//
```

Comment:
Yeah, technically, the new files you create should be copyrighted in 2016. In practice I think it's not a big deal if we miss some, but IANAL!